<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSmsPara</class>
 <widget class="QWidget" name="CSmsPara">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSmsPara</string>
  </property>
  <widget class="QPushButton" name="m_BtnDel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnOk">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnCancel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BtnAdd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_SmsTab">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>771</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>手机号码</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>描述</string>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
